### PR TITLE
✨ Metrics info card

### PR DESCRIPTION
**Pull request by MarioBatalha**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206536111880008

### CODE REVIEW

File: components/svg/pool.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const Pool: FC<SVGProps> = ({ maxWidth, maxHeight, ...props }) => (
       fillRule="evenodd"
       clipRule="evenodd"
       d="M10 5.35078V0H8V4.64922L4.51938 9H0V11H5.48062L10 5.35078ZM8 12.6492V18H10V13.3508L13.4806 9H18L18 7L12.5194 7L8 12.6492Z"
-      fill="currentColor"
+      fill="#1B1B1F"
```
Comment (git-marcopitra) [inline components/svg/pool.tsx:16]: currentColor

File: components/svg/chart.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    style={{ maxHeight, maxWidth }}
+    viewBox="0 0 24 24"
+    fill="none"
+    {...props}
+  >
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M13.9444 5.25L13.3889 5.80556V8.58333H10.0556L9.5 9.13889V11.9167H6.16667L5.61111 12.4722V16.9167H4.5V18.0278H6.16667H10.0556H13.9444H17.8333H19.5V16.9167H18.3889V5.80556L17.8333 5.25H13.9444ZM17.2778 16.9167V6.36111H14.5V9.13889V16.9167H17.2778ZM13.3889 16.9167V9.69444H10.6111V12.4722V16.9167H13.3889ZM6.72222 13.0278H9.5V16.9167H6.72222V13.0278Z"
+      fill="#1B1B1F"
```
Comment (git-marcopitra) [inline components/svg/chart.tsx:16]: currentColor

File: components/svg/usd.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { FC } from 'react';
+
+import { SVGProps } from './svg.types';
+
+const USD: FC<SVGProps> = ({ maxHeight, maxWidth, ...props }) => {
+  return (
```
Comment (git-marcopitra) [inline components/svg/usd.tsx:6]: remove the `return`

File: components/svg/usd.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      style={{ maxHeight, maxWidth }}
+      viewBox="0 0 24 24"
+      fill="none"
+      {...props}
+    >
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M13.2 2V4.4H16.8V6.8H13.2H10.8H10.2C9.72261 6.8 9.26477 6.98964 8.92721 7.32721C8.58964 7.66477 8.4 8.12261 8.4 8.6C8.4 9.07739 8.58964 9.53523 8.92721 9.87279C9.26477 10.2104 9.72261 10.4 10.2 10.4H13.8C14.9139 10.4 15.9822 10.8425 16.7698 11.6302C17.5575 12.4178 18 13.4861 18 14.6C18 15.7139 17.5575 16.7822 16.7698 17.5698C15.9822 18.3575 14.9139 18.8 13.8 18.8H13.2V21.2H10.8V18.8H7.2V16.4H10.8H13.2H13.8C14.2774 16.4 14.7352 16.2104 15.0728 15.8728C15.4104 15.5352 15.6 15.0774 15.6 14.6C15.6 14.1226 15.4104 13.6648 15.0728 13.3272C14.7352 12.9896 14.2774 12.8 13.8 12.8H10.2C9.08609 12.8 8.0178 12.3575 7.23015 11.5698C6.4425 10.7822 6 9.71391 6 8.6C6 7.48609 6.4425 6.4178 7.23015 5.63015C8.0178 4.8425 9.08609 4.4 10.2 4.4H10.8V2H13.2Z"
+        fill="#1B1B1F"
```
Comment (git-marcopitra) [inline components/svg/usd.tsx:17]: `currentColor`

File: views/metrics/index.tsx [hunk 5]
```diff
@@ -3,9 +3,9 @@ import { FC } from 'react';
 
 import Layout from '@/components/layout';
 
+import InfoCardsList from './info-card';
 import TopCoinsTable from './top-tables/top-coins-table';
 import TopPoolsTable from './top-tables/top-pools-table';
-
 const Metrics: FC = () => (
```
Comment (git-marcopitra) [inline views/metrics/index.tsx:9]: add an empty line after the imports

File: views/metrics/info-card/index.tsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+import { FC, useEffect, useState } from 'react';
+import { v4 } from 'uuid';
+
+import { formatDollars, formatMoney } from '@/utils';
+
+import { TOP_INFO_CARDS_DATA } from '../metrics.data';
+import TopInfoCards from './info-card';
+
+const InfoCardsList: FC = () => {
+  const [data, setData] = useState<any[] | null>(null);
```
Comment (git-marcopitra) [inline views/metrics/info-card/index.tsx:12]: do not use `any`, go to `sui-interface` to see how are we handling it

File: views/metrics/info-card/index.tsx [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+  useEffect(() => {
+    getMetric('get-overview').then(setData);
+  }, []);
+
+  return (
+    <Box
+      gridColumn="1/-1"
```
Comment (git-marcopitra) [inline views/metrics/info-card/index.tsx:20]: I don't know why we have this line, I think that we don't have a grid as parent, if we have it's ok

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Comment by git-marcopitra** [comment]
It's not responsive

File: views/metrics/info-card/index.tsx [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+const InfoCardsList: FC = () => {
+  const [data, setData] = useState<[]>();
+
+  useEffect(() => {
+    getMetric('get-overview').then(setData);
+  }, []);
+
+  return (
+    <Box
+      gridColumn="1/-1"
```
Comment (git-marcopitra) [inline views/metrics/info-card/index.tsx:20]: remove this line

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Comment by git-marcopitra** [comment]
image

File: views/metrics/index.tsx [hunk 9]
```diff
@@ -17,7 +18,14 @@ const Metrics: FC = () => (
     >
       Metrics
     </Typography>
-    <Box variant="container">
+    <Box
```
Comment (git-marcopitra) [inline views/metrics/index.tsx:21]: revert these changes

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Review by KipandaJr — CHANGES_REQUESTED** [review]
image

Revert that change that Marco suggest to u... put a `variant="container"` on that `Box`

**Pull request closed without merging**